extensions/windows-event-log/wel/WindowsEventLog.h

```diff
@@ -59,22 +59,19 @@ enum METADATA {
   UNKNOWN
 };
 
+using METADATA_NAMES = std::vector<std::pair<Metadata, std::string>>;
 
-// this is a continuous enum, so we can rely on the array
-
-using METADATA_NAMES = std::vector<std::pair<METADATA, std::string>>;
+nonstd::expected<std::string, std::error_code> formatEvent(EVT_HANDLE metadata, EVT_HANDLE event, EVT_FORMAT_MESSAGE_FLAGS flags) noexcept;
```

Comment:
`EVT_HANDLE` is `void*`, but I couldn't wrap it in `gsl::not_null`, because it's incompatible with `void*`. (`operator*` cannot return a reference to `void`)